keyboards/standaside/standaside.c

```diff
@@ -0,0 +1,29 @@
+
+#include "standaside.h"
+
+void matrix_init_kb(void) {
```

Comment:
Just a heads up, if you're not setting any of these functions, you can remove them.

Reply:
Neat. Good to know. I'd left them in there because I wanted a potential guide later, but they're out now.